src/ValueObject/Route.php

```diff
@@ -25,6 +25,10 @@ public function getParameters(): ?array
 
     public function addParameters(array $parameters): void
     {
+        if ($this->parameters === null) {
+            $this->parameters = $parameters;
```

Comment:
Ik snap het niet, moet dat dan niet [] zijn?

Reply:
Indien `$this->parameters` nog null is, wil dat zeggen dat de `new Route("foo")` enkel maar gedefinieerd is met een naam, en de default value `null` als parameters. Dan kunnen we nog niet [] appenden want er is nog geen array.

Reply:
amai, mijn hoofd was aan het vastlopen. Sorry.